### PR TITLE
fix unbound script source/properties

### PR DESCRIPTION
**Pull request by krzyzacy**
and pushed a new image

will fix #1397 next

### CODE REVIEW

**Comment by krzyzacy** [comment]
I'll merge to let the job run, @ixdy FYI

File: jenkins/node-image/node-runner.sh [hunk 1]
```diff
@@ -27,6 +27,5 @@ if [[ -n "${GOOGLE_APPLICATION_CREDENTIALS:-}" ]]; then
   gcloud auth activate-service-account --key-file="${GOOGLE_APPLICATION_CREDENTIALS}"
 fi
 
-
 # Run script
-${NODE_SCRIPT} ${NODE_PROPERTY}
+"${NODE_SCRIPT:-test/e2e_node/jenkins/e2e-node-jenkins.sh} ${NODE_PROPERTY:-test/e2e_node/jenkins/jenkins-ci.properties}"
```
Comment (ixdy) [inline jenkins/node-image/node-runner.sh]: this is not the fix you want.

it looks like the issue is that in `node-dockerized.sh` and `ci-kubernetes-node-kubelet-dockerized.sh` the env var is named `NODE_PROPERTIES`, not `NODE_PROPERTY`.

please fix that here and remove the defaults.

Comment (ixdy) [inline jenkins/node-image/node-runner.sh]: (in retrospect, `NODE_TEST_SCRIPT` and `NODE_TEST_PROPERTIES` are slightly better names for these vars, too.)

Comment (krzyzacy) [inline jenkins/node-image/node-runner.sh]: ouch.. good catch

**Pull request merged**